src/main/java/com/denizenscript/depenizen/bukkit/objects/towny/NationTag.java

```diff
@@ -224,6 +226,25 @@ public static void registerTags() {
             return list;
         });
 
+        // <--[tag]
+        // @attribute <NationTag.get_rank[(<rank>)]>
+        // @returns ListTag(PlayerTag)
+        // @plugin Depenizen, Towny
+        // @description
+        // Returns a list of the nation's members with a given rank.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "get_rank", (attribute, object) -> {
+            String rank = attribute.getParam();
+            ListTag list = new ListTag();
+
```

Comment:
There should not be any new lines in between the code lines. (Got to make sure it follows the rest of the style in the projects 🙂)